.github/workflows/conformance.yml

```diff
@@ -14,4 +14,4 @@ env:
   PUB_ENVIRONMENT: bot.github
 
 jobs:
-  - run: true
+  - run: false
```

Comment:
this doesn't do...anything it seems

You can see the parse error here: https://github.com/GoogleCloudPlatform/functions-framework-dart/actions/runs/383931373

Reply:
huh! ok, i'll just delete the file for now, thanks.